lua/vgui/gnbutton.lua

```diff
@@ -1,3 +1,24 @@
+--- @title:
+--- 	GNButton: <VGUI> Create a button
```

Comment:
The type, here, should be `Panel` and the description should be more detailled, something like : `A simple rounded and customisable button`